main/REPL/timerServices.md

```diff
@@ -0,0 +1,83 @@
+# Timer Service Objects
+
+`home` has two timer service objects: 
+- `localTimerService` is from the local vat.
+- `chainTimerService` is from the chain (if present). 
+
+In general, you should use the chain timer for contracts. But it's more efficient to 
+use the local timer for objects that don't need consensus or consistency. 
+For example, people might want it for testing. 
+
+Both timer services let you:
+- Get the start time of the current block. **tyg todo: Is this the case for local?**
+- Schedule `wake()` calls.
+- Remove scheduled `wake()` calls.
+- Create a repeater for scheduling of events at regular intervals. 
+
+With respect to using them, the only difference between the two timer
+services is whether you preface their identical command set with
+`localTimerService` or `chainTimerService`. However, note that
+the local and chain versions of a method may use different time bases.
+
+## `E(home.<chain or local>TimerService).getCurrentTimestamp()`
+- Returns: the current block's start time according to the local or chain clock
+
+The current block might be executed more than once in case of restart or replay.
+But each time it will start from the same state and receive the same inputs. 
+Since this is repeatable computation, the same computation can be run in various l
+ocations to cross-verify. So each time, the `currentTimeStamp` will be the same, 
```

Comment:
```suggestion
Since this is repeatable computation, the same computation can be run in various 
locations to cross-verify. So each time, the `currentTimeStamp` will be the same, 
```